### PR TITLE
Minor cleanup of the PreviewImageFragment

### PR DESCRIPTION
**Pull request by ardevd**
Just a minor cleanup of the PreviewImageFragment class. 

- Removed redundant variables
- Removed unused Override methods
- Implemented use of lambdas where applicable
- Stricter access declarations.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
:+1: looks good to me, extra credits of having moved to lambdas :1st_place_medal:

File: src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.java [hunk 1]
```diff
@@ -187,20 +187,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
         mImageView.setVisibility(View.GONE);
 
-        view.setOnClickListener(new OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                ((PreviewImageActivity) getActivity()).toggleFullScreen();
-                toggleImageBackground();
-            }
+        view.setOnClickListener(v -> {
+            ((PreviewImageActivity) getActivity()).toggleFullScreen();
```
Comment (mario) [inline src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.java]: You  have this twice. Turn it into a method.

**Comment by mario** [comment]
Small comment.

**Comment by ardevd** [comment]
@mario Another good catch. Taken care of :)

**Comment by mario** [comment]
Sorry, one more thing. DownloadFile method is a one line thing used only once. Can we just move the line where it belongs?

**Comment by ardevd** [comment]
@mario Done. Fixed a few other things while I was at it.

**Pull request merged**